packages/react/src/helpers/utils.ts

```diff
@@ -1 +1,5 @@
 export const isDevelopment = () => process.env.NODE_ENV !== 'production';
+
+export const isInputTarget = (target: unknown): target is HTMLInputElement => {
```

Comment:
you are going to hate me, but: can we rename it to `isInputEventTarget`? current name seems too generic IMO

Reply:
No problem, updated